### ISSUE TITLE
Style routes and markers according to MTA colors

### ISSUE THREAD

**Issue by talllguy** [chriswhong/mtaMarylandBusTracking#15]
Style / Cartography enhancement: The [official MTA map (pdf)](http://mta.maryland.gov/sites/default/files/MTA-Regional-Transit_0.pdf) is not great for mobile viewing, but it does have a nice color scheme for the lines. I was thinking about using some of the leaflet geojson styling to color the routes. Would that be useful?

![2015-02-13 10_08_56-mta maryland gov_sites_default_files_mta-regional-transit_0 pdf](https://cloud.githubusercontent.com/assets/1827881/6189438/8dfb77c0-b368-11e4-9151-d685aa1060cb.png)

**Comment by chriswhong** [comment]
Yes, there is actually a color attribute in the geoJSON (I think).  I think
you can style individual features using one of their attributes.
